### ISSUE TITLE
Use the command line to convert a map to a picture (Adding a headless mode)

### ISSUE THREAD

**Issue by Mfweb** [Querz/mcaselector#19]
Like this: https://github.com/TOGoS/TMCMR

**Comment by Querz** [comment]
I was thinking of adding a headless mode so you can export, delete and import chunks without opening the UI. This headless mode could also have a function to pre-render all the cache files.

If you want top-down images of your world right now from the MCA Selector you can open the `cache` folder and search through the sub folders to find images of your world, they are saved as simple png files. Keep in mind that the MCA Selector only renders images of the world that it is displaying, not all of the world.

But if you want a fancy image of your world with shading and everything I suggest to have a look at some other tools that are _meant_ to render an image of your world. The purpose of the MCA Selector is not to render a map, but to delete, export and import chunks based on certain filters.

**Comment by Querz** [comment]
@Mfweb I implemented a headless mode that will be added in the next release. You can have a look at the current pull request #28 which has a download for a preview 1.8 version.
A Documentation for the headless mode is in the new [Readme](https://github.com/Querz/mcaselector/tree/headless-mode#headless-mode).

What you probably would like to do is the following command:

```
java -jar mcaselector-1.8-all.jar --headless --mode cache --world <path-to-region-folder> --output <path-where-to-save-the-cache-files> --zoom-level 1
```
That will generate a 512x512 png image for each region of your world.

**Comment by Querz** [comment]
Published in release [1.8](https://github.com/Querz/mcaselector/releases/tag/1.8).
